x-pack/plugins/threat_intelligence/public/modules/indicators/components/indicators_barchart/indicators_barchart.tsx

```diff
@@ -52,6 +53,7 @@ export const IndicatorsBarChart: VFC<IndicatorsBarChartProps> = ({
           showLegend
           showLegendExtra
```

Comment:
Took a look at the charts documentation. It's quite weird that the legend is controlled only through the Settings component. My suggestion is to remove the `showLegendExtra` from the settings because values in the legend in my opinion don't bring any value, only confusion. Tbh the current behaviour of the legend looks to my more like a UX bug than a feature, especially when combined with a tooltip on hower on each bar. Does it makes sense to consult with the team owning the chart library and ask how would they suggest to implement such a feature - showing the accumulated values in the legend

Reply:
found this issue that is related to the problem https://github.com/elastic/elastic-charts/issues/561

Reply:
yeah good point on removing the `showLegendExtra`, I really don't see what value it provide at the moment as we have the tooltip that's way more convenient.

I'll reach out to the `charts` team and ask them about the accumulated values in legend!